lib/internal/Magento/Framework/composer.json

```diff
@@ -22,7 +22,8 @@
         "ext-xsl": "*",
         "symfony/process": "~2.1",
         "zendframework/zend-stdlib": "~2.4.6",
-        "zendframework/zend-http": "~2.4.6"
+        "zendframework/zend-http": "~2.4.6",
+        "magento/zendframework1": "~1.13.0"
```

Comment:
Please change constraint to match root `composer.json`: https://github.com/magento/magento2/blob/2.1-develop/composer.json#L34, amend commit and force push.